vcd/resource_vcd_nsxt_network_dhcp_test.go

```diff
@@ -43,20 +45,28 @@ func TestAccVcdOpenApiDhcpNsxtRouted(t *testing.T) {
 	}
 
 	// This case is specific for VCD 10.3.1 onwards since dns servers are not present in previous versions
-	var configText2 string
+	var configText3 string
+	var configText4 string
```

Comment:
nit, but could we just use the `:=` operator for readability?

Reply:
Fixed